scripts/index_d_ts

```diff
@@ -667,6 +667,87 @@ declare module "SendbirdUIKitGlobal" {
     editDisabled?: boolean;
   };
 
+  /**
+   * OpenChannelList
+   */
+  export interface UserFilledOpenChannelListQuery {
+    customTypes?: Array<string>;
+    includeFrozen?: boolean;
+    includeMetaData?: boolean;
+    limit?: number;
+    nameKeyword?: string;
+    urlKeyword?: string;
+  }
+
+  export type OpenChannelListFetchingStatus = 'EMPTY' | 'FETCHING' | 'DONE' | 'ERROR';
+  export type CustomUseReducerDispatcher = (props: { type: string, payload: any }) => void;
+  export type OnOpenChannelSelected = (channel: OpenChannel, e?: React.MouseEvent<HTMLDivElement | unknown>) => void;
+  export type FetchNextCallbackType = (callback: (channels?: Array<OpenChannel>, err?: SendbirdError) => void) => void
+
+  export interface OpenChannelListProviderProps {
+    className?: string;
+    children?: React.ReactElement;
+    queries?: { openChannelListQuery?: UserFilledOpenChannelListQuery };
+    onChannelSelected?: OnOpenChannelSelected;
+  }
+
+  export interface OpenChannelListUIProps {
+    renderHeader?: () => React.ReactElement;
+    renderChannelPreview?: (props: RenderChannelPreviewProps) => React.ReactElement;
```

Comment:
RenderChannelPreviewProps -> in this file returns groupChannel, no?

Reply:
better have `RenderOpenChannelPreviewProps `

Reply:
good catch